spec/book_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../classes/book'
+require 'rspec'
+
```

Comment:
Fantastic work adding adequate unit tests for all your classes 💯 
Thank you for showing dedication in adding well-structured, well-organized, and comprehensive tests for all the class methods. 🚀 

![image](https://github.com/Abdulwhbhussain/OOP-school-library/assets/38088818/0f3c33fb-4709-4b06-bd1e-cc23c4a003d1)


Overall, everything looks great, all the tests pass and the code is clean making use of the "describe" and "it" blocks to categorize each test. 🥂

Reply:
Thanks for the feedback!